account_payment_partner/tests/test_account_payment_partner.py

```diff
@@ -0,0 +1,588 @@
+# Copyright 2017 ForgeFlow S.L.
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, fields
+from odoo.exceptions import UserError, ValidationError
+from odoo.fields import Date
+from odoo.tests.common import Form, TransactionCase, tagged
```

Comment:
```suggestion
from odoo.tests import Form, TransactionCase, tagged
```
`DeprecationWarning: Since 18.0: odoo.tests.common.Form is deprecated, use odoo.tests.Form`